addons/medical/scripts/Game/ACE_Medical/Components/Damage/SCR_CharacterDamageManagerComponent.c

```diff
@@ -95,6 +97,19 @@ modded class SCR_CharacterDamageManagerComponent : SCR_DamageManagerComponent
 		
 		return false;
 	}
+	
+	//------------------------------------------------------------------------------------------------
+	//! Returns true if at least one physical hit zone can be healed using the medkit limited by the medical kit maximum heal
+	bool ACE_Medical_MedkitUsable()
```

Comment:
This method has basically a duplicate of `ACE_Medical_CanBeHealed`. Remove it and instead expand `ACE_Medical_CanBeHealed` by introducing a boolean argument for whether to consider medical facilities.

Reply:
Method replaced with altered method, referenced updated.